src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUnaryOperatorExpression.cs

```diff
@@ -3545,7 +3545,7 @@ bool F(bool f)
 
 
         [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
-        [Fact]
+        [Fact(Skip = "PROTOTYPE(BangBang)")]
```

Comment:
What's wrong with this test?

Reply:
It works now with the new parsing method, removed the skip.